### PR TITLE
Update dcos-monitoring 1.3.1

### PR DESCRIPTION
**Pull request by janisz**
## Jira Ticket
https://jira.d2iq.com/browse/D2IQ-71300

## Description of changes being made

Release of 1.3.1

## Checklist
- [x] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [x] Test all commands and procedures, if applicable.
- [x] Create your PR against `staging`, not `master`. 
- [x] Provide an estimated date for deploying the doc change. Note: Improvements or fixes can be merged ASAP. 
- [x] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.
- [x] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/release-notes/index.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes
+title: Release Notes
+menuWeight: 40
+excerpt: New features, updates, and known limitations
+render: mustache
+model: ../data.yml
+---
+
```
Comment (cmastr) [inline pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/release-notes/index.md:10]: Must have a date for release.

File: pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/release-notes/index.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+* alertmanager: Add a package option for git ref name.
+* Add an option to configure prometheus remote write and remote read.
+
+## Updates
+
+* Updated SDK to v0.57.3.
+
+## Notes
+
+* A `pod replace` operation is required if there are existing deployments of older versions on DC/OS 2.0. See the SDK release notes for details at <https://github.com/mesosphere/dcos-commons/releases/tag/0.57.3>.
```
Comment (cmastr) [inline pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/release-notes/index.md:31]: DC/OS 2.0?

File: pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/release-notes/index.md (hunk 1 above)
Comment (janisz) [reply pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/release-notes/index.md:10]: Can you link an example?

File: pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/release-notes/index.md (hunk 2 above)
Comment (janisz) [reply pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/release-notes/index.md:31]: It's changelog for 1.3.0
https://github.com/mesosphere/dcos-docs-site/blob/staging/pages/mesosphere/dcos/services/dcos-monitoring/1.3.0/release-notes/index.md#notes

File: pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/release-notes/index.md (hunk 1 above)
Comment (janisz) [reply pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/release-notes/index.md:10]: https://github.com/mesosphere/dcos-docs-site/pull/3113/files#diff-62e96a3337cf3ddceb18058a74c7e938R11

**Comment by janisz** [comment]
@pierrebeitz How can I make Vale happy? It's failing on URLs :(

File: .github/styles/proselint/Annotations.yml [hunk 3]
```diff
@@ -6,4 +6,3 @@ tokens:
   - XXX
   - FIXME
   - TODO
-  - NOTE
```
Comment (janisz) [inline .github/styles/proselint/Annotations.yml:9]:
```bash
git grep NOTE | wc -l
1932
```

Comment (pierrebeitz) [reply .github/styles/proselint/Annotations.yml:9]: thanks! we'll add an according exclude.

**Comment by janisz** [comment]
https://github.com/errata-ai/vale/issues/231

**Comment by pierrebeitz** [comment]
@janisz we're still in the process of finding a good ruleset for vale and PRs don't necessarily need to be green just yet.

thanks for going the extra mile and making suggestions to the vocabulary!

**Comment by janisz** [comment]
@cmastr It's ready to merge

**Comment by janisz** [comment]
> thanks for going the extra mile and making suggestions to the vocabulary!

No problem. I think we should be strict and do not allow changes that fails linting. I takes me some time to fix all issues and tune vale but I think if we spend this extra time now we will get better docs in the future.

**Comment by janisz** [comment]
Close in favour of https://github.com/mesosphere/dcos-docs-site/pull/3186

**Pull request closed without merging**